ingestion/src/metadata/ingestion/source/pipeline/domopipeline.py

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
can you add the copyright comment in all new files

Reply:
Yes will add.